### PR TITLE
Vikram ganti caption add new

### PR DESCRIPTION
**Pull request by vikram-ganti**
This pull request

### CODE REVIEW

File: _slides/02-beardofedu.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+GitHub - Hello Kellogg's
```
Comment (Pilo1961) [inline _slides/02-beardofedu.md:6]: Nice

File: _slides/18-vikram-ganti.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello.....
```
Comment (vikram-ganti) [inline _slides/18-vikram-ganti.md:6]: good

**Review by vikram-ganti — COMMENTED** [review]
ad

File: _slides/02-beardofedu.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (Pilo1961) [inline _slides/02-beardofedu.md:3]:
```suggestion
title: "Welcome to vikram-ganti slide!"
```

**Review by Pilo1961 — APPROVED** [review]
summary of review comments

**Review by vikram-ganti — COMMENTED** [review]
thanks

**Review by vikram-ganti — COMMENTED** [review]
nice

**Pull request merged**